### DISCUSSION TITLE
[help] Can I force a pipeline to be up to date without running it?

### DISCUSSION THREAD

**Discussion by riprobins** [ropensci/targets#1084] [Help] [answered]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

My pipeline takes around 20hrs to complete, and I have a function I'm using right at the start.  Somehow I changed this, realised, and then altered it back, but my pipeline now wants to run from the beginning.  I really dont want to rerun everything, so is there a way to make all existing targets appear to be completed, so I can continue with the rest of the pipeline?

**Comment by wlandau** [comment, marked as answer]
Please have a look at cues: https://docs.ropensci.org/targets/reference/tar_cue.html. You can set `cue = tar_cue(depend = FALSE)` either for an individual target or globally with `tar_option_set()`.

**Reply by riprobins** [reply]
Awesome, thank you
